Purchases/Networking/Caching/CallbackCache.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  CallbackCache.swift
+//
+//  Created by Joshua Liebowitz on 11/18/21.
+
+import Foundation
+
+class CallbackCache<T> where T: CachableCallback {
```

Comment:
What do you think about adding some docs here? At first sight it's hard to see what its purpose is.

Reply:
✅